src/main/kotlin/com/server/jobboxapp/repository/BlogRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.server.jobboxapp.repository
+
+import com.server.jobboxapp.entity.BlogPost
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
+
+@Repository
+interface BlogRepository : JpaRepository<BlogPost, Long>{
+
+    @Transactional
+    @Modifying
+    @Query("UPDATE BlogPost b SET b.headline = ?2 WHERE b.id = ?1")
```

Comment:
Czemu nie korzytasz z JPA ?

Reply:
Miałem złe doświadczenia z JPA kiedyś i wolę pisać native query, ale to można zmienić :)